e2e/api_test.go

```diff
@@ -14,6 +14,7 @@ import (
 
 	"github.com/hashicorp/consul-terraform-sync/api"
 	"github.com/hashicorp/consul-terraform-sync/testutils"
+	ctsTestClient "github.com/hashicorp/consul-terraform-sync/testutils/cts"
```

Comment:
I'm not very satisfied with the package name, lmk if any others have thoughts here.

Reply:
My only thought to throw out would be to name the package something like `ctsTest` and rename `StartCTS()` to `StartClient()`. This sort of leans on the idea that we might have more cts-related utils that aren't test-client related. I also was thinking the `StartCTS()`'s usage of "cts" could be perceived as redundant.

Just threw it out in case! ctsTestClient works for me and we can rename down the line as it evolves.